netops_demo/golang/src/github.com/v3io/demos/functions/ingest/ingest.go

```diff
@@ -227,6 +241,55 @@ func ingestMetricSamplesToTSDB(context *nuclio.Context,
 	return nil
 }
 
+func ingestMetricSamplesToKV(context *nuclio.Context,
+	kvContainer *v3io.Container,
+	emitterLabels map[string]interface{},
+	metricName string,
+	samples *metricSamples) error {
+
+	baseKVPath := os.Getenv("INGEST_V3IO_KV_PATH")
+	metricPath := baseKVPath + "/" + metricName
+	responseChannel := make(chan *v3io.Response, 1) // Only one response per PutItems
+
+	items := make(map[string]map[string]interface{}, len(samples.Timestamps))
+	for index, timestamp := range samples.Timestamps {
+		key := strconv.FormatInt(timestamp, 10) + "_" + emitterLabels["device"].(string)
+		items [key] = map[string]interface{}{
+			string("timestamp"): string(timestamp),
+			string("alert"):     string(samples.Alerts[index]),
+			string("IsError"): int(samples.IsError[index]),
+		}
+		for k, v := range emitterLabels {
+			items [key][k] = v
+		}
+	}
+
+	// TODO: Why Emitter Labels doesn't hold device ID? Only company name / site
+	_, PutItemErr := kvContainer.PutItems(&v3io.PutItemsInput{
```

Comment:
1. No need for the response channel. Just use `kvContainer.Sync.PutItems` - it's blocking
2. `putItemErr` (lowercase)